components/Converters/samples/ColorToDisplayNameConverterSample.xaml

```diff
@@ -0,0 +1,32 @@
+﻿<!--  Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information.  -->
+<Page x:Class="ConvertersExperiment.Samples.ColorToDisplayNameConverterSample"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:converters="using:CommunityToolkit.WinUI.Converters"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:local="using:ConvertersExperiment.Samples"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+      mc:Ignorable="d">
+
+    <Page.Resources>
+        <converters:ColorToDisplayNameConverter x:Key="ColorToDisplayNameConverter" />
+    </Page.Resources>
+
+    <StackPanel HorizontalAlignment="Left"
+                Spacing="16">
+
+        <ColorPicker x:Name="colorPicker"
+                     Height="64"
```

Comment:
I guess this is meant to be cut-off? It looks a little odd with the shape:

![image](https://github.com/CommunityToolkit/Windows/assets/24302614/a1a8a41e-332e-441c-96fa-1335dc65d2a1)

Thoughts on if we return a "Not Supported" string instead of throwing an exception?